### PR TITLE
Explicit return types

### PR DESCRIPTION
**Pull request by ericcater**
setting explicit return types for servicer/resolver

TODO set it only on backend files

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -24,13 +24,35 @@
...
       "extends": ["plugin:@nrwl/nx/javascript"],
       "rules": {}
     },
+    {
+      "files": ["*.resolver.ts", "*.service.ts"],
+      "parser": "@typescript-eslint/parser",
+      "plugins": ["@typescript-eslint"],
+      "rules": {
+        //TODO would like to make up this to error eventually
+        "@typescript-eslint/explicit-function-return-type": "warn"
```
Comment (yurisim) [inline .eslintrc.json:42]: Causes issues when we use complicated return types through include or select in the prisma level. Can you exclude the frontend service files?

File: .eslintrc.json [hunk 2]
```diff
@@ -24,13 +24,35 @@
     {
       "files": ["*.ts", "*.tsx"],
       "extends": ["plugin:@nrwl/nx/typescript"],
-      "rules": {}
+      "rules": {
+        "no-return-await": "warn"
+      }
```
[the remark this answers is not in the document]
Comment (ericcater) [reply .eslintrc.json:29]: it is not.

File: .eslintrc.json (hunk 1 above)
Comment (ericcater) [reply .eslintrc.json:42]: It's a warning right now, so ignore it on that for now. But I'd like to move away from those queries. In part for better return types, but also for more reusable code, etc. If batching is still a concern, the best solution is likely implementing a custom dataloader.

**Comment by ericcater** [comment]
Would like to rework how it matches files. The better way is implemented on #162 .

**Pull request closed without merging**